pennylane/operation.py

```diff
@@ -481,66 +481,74 @@ def matrix(self, wire_order=None):
         return expand_matrix(canonical_matrix, wires=self.wires, wire_order=wire_order)
 
     @classmethod
-    def _eigvals(cls, *params):
-        """Eigenvalues of the operator.
+    def compute_eigvals(cls, *params, **hyperparams):
+        """Canonical eigenvalues of the operator in the computational basis.
 
-        This is a *class method* that should be defined for all
-        new operations and observables that returns the eigenvalues
-        of the operator.
+        The canonical eigenvalues refer to the textbook matrix representation that does not consider wires.
+        Implicitly, this assumes that the wires of the operator correspond to the global wire order.
+
+        This class method allows eigenvalues to be computed
+        directly without instantiating the operator first.
+
+        The default implementation relies on the presence of the
+        :attr:`~.Operator.compute_matrix` method. To return the eigenvalues of *instantiated* operators,
+        please use the :attr:`~.Operator.eigvals()` method instead.
```

Comment:
```suggestion
        :meth:`~.Operator.compute_matrix` method. To return the eigenvalues of *instantiated* operators,
        please use the :meth:`~.Operator.eigvals()` method instead.
```

Reply:
although now I worry I never noticed this in other PRs 😆

Reply:
No worries, I am making a search/replace and sneak it into this PR!